frontend/src/app/accountCreation/LoadingCard/LoadingCard.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Story, Meta } from "@storybook/react";
+
+import { LoadingCard } from "./LoadingCard";
+
+export default {
+  title: "App/Account set up/Loading card",
+  component: LoadingCard,
+  argTypes: {},
+  args: {
+    message: "Verifying security code ...",
```

Comment:
```suggestion
    message: "Verifying security code …",
```

Reply:
Changing from 3 periods to `&hellip;` - should this be in the component rather than added to each use of the component since they all use the same pattern?

Reply:
Yeah good idea, I can def move it into the component

Reply:
Fixed in 5993c15